src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Tooltip/DefaultLSPTagHelperTooltipFactory.cs

```diff
@@ -50,9 +50,8 @@ internal class DefaultLSPTagHelperTooltipFactory(ISnapshotResolver snapshotResol
             }
 
             var tagHelperType = descriptionInfo.TagHelperTypeName;
-            var reducedTypeName = ReduceTypeName(tagHelperType);
             StartOrEndBold(descriptionBuilder, markupKind);
-            descriptionBuilder.Append(reducedTypeName);
+            descriptionBuilder.Append(tagHelperType);
```

Comment:
Perhaps when the unit test fixes are in it will be obvious, but I would be interested to see a before and after screenshot to understand what this looks like in reality.

One thing I'm wondering is whether the whole type name should be bold. I wonder if we can keep the reduced type name, and have that in bold, but include the full namespace as well, just not in bold.

ie, it used to do:

> **Routes**

with this change it will do:

> **Microsoft.AspNet.Components.Routes**

but what about if we did

> Microsoft.AspNet.Components.**Routes**

I think visually that might match the VS behaviour better.

Reply:
The bold text on the reduced name makes more sense. I can fix that and include a before and after picture